### PR TITLE
Fixing Typo - removing static in build command

### PR DESCRIPTION
**Pull request by rahilzebpay**
## Type of change

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🛠 New feature (non-breaking change which adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [ ] 🔨 New or improved tests for existing code
- [ ] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [ ] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [ ] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.

## Testing checklist

- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Two steps remain, then:
 2.  Actually building the Cardano SL node (or, most likely, simply obtaining it
     from the IOHK's binary caches) can be performed by building the attribute `cardano-sl-static`:
```
Comment (akegalj) [inline docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md]: @rahilzebpay we should update description as well here

**Comment by akegalj** [comment]
@jmitchell 
this build command seems stale - at least from my quick skim over `release.nix`

File: docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md (hunk 1 above)
Comment (rahilzebpay) [reply docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md]: @akegalj  Done!

Comment (akegalj) [reply docs/how-to/build-cardano-sl-and-daedalus-from-source-code.md]: Hm, after another look at the line:
> Actually building the Cardano SL node...

I believe the correct binary that should be triggered here is `cardano-sl-node-static` (after a look at `release.nix`).

@jmitchell what is a difference between building `cardano-sl` and `cardano-sl-node-static` ? I believe the documentation is trying to give an example how to build a cardano node and later gives more examples on how to build other binaries (the later part of the documentation is also incorrect. For example there is a mention of `cardano-sl-wallet-static` which doesn't exist any more in our `release.nix`)

**Comment by jcmincke** [comment]
@akegalj : can  I close this PR?

**Comment by akegalj** [comment]
@jcmincke 
> can I close this PR?

yes.

**Pull request closed without merging**